source/board/max32690evkit.c

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file    max32690evkit.c
+ * @brief   board ID for the Analog Devices Inc.'s MAX32690EVKIT
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2023, Analog Devices, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "target_family.h"
+#include "target_board.h"
+
+const board_info_t g_board_info = {
+	.info_version = kBoardInfoVersion,
+    .board_id = "0426",
+    .family_id = kMaxim_MAX3266X_FamilyID,
+    .flags = kEnablePageErase,
+    .target_cfg = &target_device,
+    .daplink_url_name = "MAX32690HTM",
+    .daplink_target_url = "https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/MAX32690EVKIT.html",
+    .board_vendor = "Analog Devices Inc.",
+    .board_name = "MAX32690EVKIT",
```

Comment:
`.board_vendor` and `.board_name` need to match values in CMSIS pack (see note in https://arm-software.github.io/CMSIS_5/DAP/html/group__DAP__Info.html). Based on `Maxim.MAX32690.pdsc` from https://www.mxim.net/microcontroller/pack/Maxim.MAX32690.1.3.0.pack, they should be respectively `"Maxim"` and `"MAX32690 EvKit"`.

Reply:
Thanks, updated.

Reply:
This change is still pending.

Reply:
Updated as "Maxim" and "MAX32690 EvKit".
Thanks.